master/getting-started/kubernetes/installation/calico.md

```diff
@@ -47,7 +47,7 @@ datastore type and number of nodes.
 1. Download the {{site.prodname}} networking manifest for the Kubernetes API datastore.
 
    ```bash
-   curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -O
+   curl {{site.url}}/{{page.version}}/manifests/calico.yaml -O
```

Comment:
I don't think this is the right fix - this changes the manifest that is downloaded to be a different one.

We could either update this line below: https://github.com/projectcalico/calico/pull/2674/files#diff-4206179465750ddee2eb8085ba8f2fb2R85

Or change to this:

```
curl {{site.url}}/{{page.version}}/manifests/calico-typha.yaml -o calico.yaml
```

Reply:
The discussion is here: https://tigera.slack.com/messages/C0FQH92CA/convo/C2PQSHPCG-1560870200.169500/?cdn_fallback=1

I'm confused then.

Reply:
Updated curl command.